pkg/controller/common/version/version.go

```diff
@@ -13,6 +13,33 @@ import (
 	"github.com/pkg/errors"
 )
 
+// supported Stack versions. See https://www.elastic.co/support/matrix#matrix_compatibility
+var (
+	SupportedAPMServerVersions        = MinMaxVersion{Min: From(6, 2, 0), Max: From(8, 99, 99)}
+	SupportedElasticsearchVersions    = MinMaxVersion{Min: From(6, 8, 0), Max: From(8, 99, 99)}
```

Comment:
We don't support 7.0 but only > 7.1 because this is the version that made TLS available under the basic license. So because we also support 6.8 this needs to be a non-contiguous range (not sure if that's the right word)

Reply:
Good catch! This is a leftover line that I forgot to remove. I didn't touch the Elasticsearch validation code in this PR because I noticed this problem as well. It's a refactoring that I wanted to do in a subsequent PR but I forgot to mention it. The [logic for Elasticsearch](https://github.com/elastic/cloud-on-k8s/blob/5ae992381ac44e31dc4c0bc72823acfd081ae09d/pkg/controller/elasticsearch/version/supported_versions.go#L19) is unchanged.